test/integration/db_integration_test.go

```diff
@@ -56,13 +62,11 @@ func (a *dbIntegrationSuite) TestGetRecordsFromDb() {
 	reply, err := stream.CloseAndRecv()
 	require.NoError(a.T(), err)
 	require.True(a.T(), reply.Ok)
-
 	require.Len(a.T(), a.c.Importer.GetSavedRecords(), recordsCount)
 
 	ctx := context.Background()
-	extractorMn := extractor.NewMainNetExtractor(100, a.c.ExporterClient)
 
-	jetDrops := extractorMn.GetJetDrops(ctx)
+	jetDrops := a.be.Extractor().MainJetDropsChan
```

Comment:
Why make `MainJetDropsChan` public and use it here?

Reply:
resolved after merge with master